ckan/lib/dictization/__init__.py

```diff
@@ -45,6 +45,8 @@ def table_dictize(obj, context, **kw):
             result_dict[name] = value
         elif isinstance(value, int):
             result_dict[name] = value
+        elif isinstance(value, long):
```

Comment:
Leave it, sounds like the right approach anyway. But add a quick note on the changelog file for the next version saying that the `resource['size']` property will change from a string to a number, in case someone relies on it.